ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -124,17 +129,29 @@ object LedgerApiTestTool {
         sys.exit(1)
       })
 
+    val included_p =
+      if (config.allTests) Tests.all(config).map(_.name).toSet
+      else if (config.included.isEmpty) Tests.default.map(_.name).toSet
+      else config.included
+    val shouldRun: String => Boolean = name => {
+      included_p.filter(name.startsWith(_)).nonEmpty && config.excluded
+        .filter(name.startsWith(_))
+        .isEmpty
+    }
+
     val runner =
       if (performanceTestsToRun.nonEmpty)
         newLedgerSuiteRunner(
           config,
           performanceTestsToRun,
+          _ => true,
```

Comment:
maybe worth using a named argument here?

Reply:
Renamed to `caseName`, in both this one and the non-perf version.